Detectors/MUON/MCH/Workflow/src/clusters-sink-workflow.cxx

```diff
@@ -206,28 +208,28 @@ class ClusterSinkTask
   bool mDoDigits = true;         ///< whether or not we deal with digits
 };
 
-// add workflow options, note that customization needs to be declared before
-// including Framework/runDataProcessing
-void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+//_________________________________________________________________________________________________
+void customize(std::vector<ConfigParamSpec>& workflowOptions)
 {
-  std::vector<o2::framework::ConfigParamSpec> options{
-    {"global", o2::framework::VariantType::Bool, false, {"read clusters with positions expressed in global reference frame"}},
-    {"no-digits", o2::framework::VariantType::Bool, false, {"do not look for digits"}},
-  };
-  std::swap(workflowOptions, options);
+  /// add workflow options. Note that customization needs to be declared before including Framework/runDataProcessing
```

Comment:
Hi, this is of course complete nitpicking compared to the very useful fixes you introduce here, but I'd have two stylistic comments :  
- I'm not keen on the //________________ lines, but that's probably personal 
- less personal : the functions comments should be before the functions, not inside (see https://rawgit.com/AliceO2Group/CodingGuidelines/master/comments_guidelines.html?showone=Functions#Functions), shouldn't they ?

Reply:
I like the //_____ lines because it helps me to visualize the code. ;-)
For the comments, indeed, I missed this guideline, so I kept doing what I was used to.